Description: accept older minor version of crate libtest-mimic
Author: Jonas Smedegaard <dr@jones.dk>
Bug-Debian: https://bugs.debian.org/1127572
Forwarded: not-needed
Last-Update: 2026-04-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -121,7 +121,7 @@
 indexmap = { version = "2.7.0", default-features = false }
 indoc = "2.0.5"
 leb128fmt = { version = "0.1.0", default-features = false }
-libtest-mimic = "0.8.1"
+libtest-mimic = ">= 0.7.3, <= 0.8.1"
 log = "0.4.17"
 num_cpus = "1.13"
 pretty_assertions = "1.3.0"
